app/views/simple_captcha/_simple_captcha.haml

```diff
@@ -1,4 +1,2 @@
-.control-label
-  = simple_captcha_options[:image]
-.controls
-  = simple_captcha_options[:field]
\ No newline at end of file
+= simple_captcha_options[:image]
+= simple_captcha_options[:field]
```

Comment:
@jaywink it's possible to put a class on the captcha here. But the problem is not to select it to css, it's which style to apply... I can align it right making it floating, and as the border between the image and the input is on the input (and we will never have focus on the image), this would work. But then we have a floating element, so the top and bottom borders are hidden by the container div...